integration/tests/cook/test_basic.py

```diff
@@ -994,16 +994,60 @@ def check_unique_constraint(unscheduled_jobs):
         finally:
             util.kill_jobs(self.cook_url, [uuids])
 
+    def test_balanced_host_constraint_cannot_place(self):
+        state = util.get_mesos_state(self.mesos_url)
+        num_hosts = len(state['slaves'])
+        if num_hosts > 10:
+            # Skip this test on large clusters
+            return
+        minimum_hosts = num_hosts + 1
```

Comment:
Nitpick: Do we need a separate `minimum_hosts` variable?

Reply:
We don't need one, but I think it adds to clarity

Reply:
okay